src/open-api/from-open-api.ts

```diff
@@ -11,6 +11,12 @@ const supportedHttpMethods = Object.keys(
   http,
 ) as unknown as SupportedHttpMethods
 
+export type MapOperationFunction = (
+  url: string,
```

Comment:
Let's expose the whole parsed `document` of the spec as well. That way, developers can access any information anywhere to help them filter or map operations.

Reply:
e6a83f2df69a984ce840a9c24571720ce2f5c871